<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>main_field</class>
 <widget class="QMainWindow" name="main_field">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1360</width>
    <height>748</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1360</width>
    <height>748</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1360</width>
    <height>748</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="world_map">
    <property name="geometry">
     <rect>
      <x>-70</x>
      <y>0</y>
      <width>1151</width>
      <height>731</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>F:/–§–æ—Ç–æ/–∫–∞—Ä—Ç–∞_–º–∏—Ä–∞.jpg</pixmap>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_russia">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>152</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Sans Serif</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(255, 255, 255, 0);
border: 3px solid #000000;
border-radius: 5px;
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_germany">
    <property name="geometry">
     <rect>
      <x>506</x>
      <y>224</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(255, 255, 255, 0);
border: 3px solid #000000;
border-radius: 5px;
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_sweden">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>174</y>
      <width>31</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(255, 255, 255, 0);
border: 3px solid #000000;
border-radius: 5px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_spain">
    <property name="geometry">
     <rect>
      <x>468</x>
      <y>267</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(255, 255, 255, 0);
border: 3px solid #000000;
border-radius: 5px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_canada">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>187</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(255, 255, 255, 0);
border: 3px solid #000000;
border-radius: 5px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_brazil">
    <property name="geometry">
     <rect>
      <x>314</x>
      <y>400</y>
      <width>61</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(255, 255, 255, 0);
border: 3px solid #000000;
border-radius: 5px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_china">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>272</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(255, 255, 255, 0);
border: 3px solid #000000;
border-radius: 5px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_australia">
    <property name="geometry">
     <rect>
      <x>824</x>
      <y>450</y>
      <width>81</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(255, 255, 255, 0);
border: 3px solid #000000;
border-radius: 5px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_india">
    <property name="geometry">
     <rect>
      <x>693</x>
      <y>310</y>
      <width>41</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(255, 255, 255, 0);
border: 3px solid #000000;
border-radius: 5px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_japan">
    <property name="geometry">
     <rect>
      <x>878</x>
      <y>268</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(255, 255, 255, 0);
border: 3px solid #000000;
border-radius: 5px; </string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="search_field">
    <property name="geometry">
     <rect>
      <x>1090</x>
      <y>30</y>
      <width>251</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="search_lbl">
    <property name="geometry">
     <rect>
      <x>1090</x>
      <y>10</y>
      <width>191</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>üîé –ü–æ–∏—Å–∫</string>
    </property>
   </widget>
   <widget class="QTableView" name="result_field">
    <property name="geometry">
     <rect>
      <x>1090</x>
      <y>90</y>
      <width>251</width>
      <height>231</height>
     </rect>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::NoEditTriggers</set>
    </property>
   </widget>
   <widget class="QPushButton" name="mainBtn_russia">
    <property name="geometry">
     <rect>
      <x>729</x>
      <y>151</y>
      <width>133</width>
      <height>43</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #ff593c;
font: 13pt &quot;MS Reference Sans Serif&quot;;</string>
    </property>
    <property name="text">
     <string>–†–æ—Å—Å–∏—è</string>
    </property>
   </widget>
   <widget class="QPushButton" name="mainBtn_china">
    <property name="geometry">
     <rect>
      <x>719</x>
      <y>271</y>
      <width>77</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #808000;
font: 11pt &quot;MS Reference Sans Serif&quot;;</string>
    </property>
    <property name="text">
     <string>–ö–∏—Ç–∞–π</string>
    </property>
   </widget>
   <widget class="QPushButton" name="mainBtn_germany">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>220</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #9ACD32;
font: 8pt &quot;MS Reference Sans Serif&quot;;</string>
    </property>
    <property name="text">
     <string>–ì–µ—Ä–º–∞–Ω–∏—è</string>
    </property>
   </widget>
   <widget class="QPushButton" name="mainBtn_australia">
    <property name="geometry">
     <rect>
      <x>822</x>
      <y>450</y>
      <width>83</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:#7B68EE;
font: 10pt &quot;MS Reference Sans Serif&quot;;</string>
    </property>
    <property name="inputMethodHints">
     <set>Qt::ImhNone</set>
    </property>
    <property name="text">
     <string>–ê–≤—Å—Ç—Ä–∞–ª–∏—è</string>
    </property>
   </widget>
   <widget class="QPushButton" name="mainBtn_india">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>309</y>
      <width>51</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #9ACD32;
font: 8pt &quot;MS Reference Sans Serif&quot;;</string>
    </property>
    <property name="text">
     <string>–ò–Ω–¥–∏—è</string>
    </property>
   </widget>
   <widget class="QPushButton" name="mainBtn_japan">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>262</y>
      <width>61</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #808000;
font: 9pt &quot;MS Reference Sans Serif&quot;;</string>
    </property>
    <property name="text">
     <string>–Ø–ø–æ–Ω–∏—è</string>
    </property>
   </widget>
   <widget class="QPushButton" name="mainBtn_sweden">
    <property name="geometry">
     <rect>
      <x>505</x>
      <y>172</y>
      <width>61</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #4169E1;
font: 9pt &quot;MS Reference Sans Serif&quot;;</string>
    </property>
    <property name="text">
     <string>–®–≤–µ—Ü–∏—è</string>
    </property>
   </widget>
   <widget class="QPushButton" name="mainBtn_canada">
    <property name="geometry">
     <rect>
      <x>134</x>
      <y>186</y>
      <width>81</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #9370DB;
font: 11pt &quot;MS Reference Sans Serif&quot;;</string>
    </property>
    <property name="text">
     <string>–ö–∞–Ω–∞–¥–∞</string>
    </property>
   </widget>
   <widget class="QPushButton" name="mainBtn_brazil">
    <property name="geometry">
     <rect>
      <x>306</x>
      <y>399</y>
      <width>75</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #808000;
font: 9pt &quot;MS Reference Sans Serif&quot;;</string>
    </property>
    <property name="text">
     <string>–ë—Ä–∞–∑–∏–ª–∏—è</string>
    </property>
   </widget>
   <widget class="QPushButton" name="mainBtn_spain">
    <property name="geometry">
     <rect>
      <x>458</x>
      <y>262</y>
      <width>51</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #FA8072;</string>
    </property>
    <property name="text">
     <string>–ò—Å–ø–∞–Ω–∏—è</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1360</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
